CONP_DATS_fields.md

```diff
@@ -15,7 +15,8 @@ The information will be used to create your resource landing page on the CONP po
 |distributions - format|**REQUIRED**. Primary data files format (*example*: csv, nifti, txt, fasta).|
 |distributions - size|**REQUIRED**. Total size of the dataset.|
 |distributions - unit|**REQUIRED**. Unit in which the size is measured.(*example*: GB)|
-|distributions - access|**REQUIRED**. Web address where the original dataset can be found.|
+|distributions - access - landingPage|**REQUIRED**. Web address where the original dataset can be found.|
+|distributions - access - annotation|Value must be one of "public", "registered" or "private". When this field is absent the value will be treated as "public".|
```

Comment:
There is just this part. Should we consider every datasets as `Private` when not present as a fail safe as discussed on other channel?

Reply:
`Annotation` should be in an `extraProperties`

Reply:
I would be in favour of that provided there is general agreement on this point, I think it was @glatard who suggested public as default but I may be mistaken.

Reply:
Sorry, that is an error in the .md and I shall update the file.  The information about access goes in _distributions->access->authorizations_ rather than _extraProperties->annotation_.

Reply:
`authorizations` need an `annotation`

Reply:
I'm sorry, I am not clear what you are asking for; could you expand on your last comment ?  I'm following https://github.com/CONP-PCNO/conp-dataset/pull/198.

Reply:
Exactly, the `authorizations` field require an array of `annotation` and in an `annotation`, you can have a `value` just like in https://github.com/CONP-PCNO/conp-dataset/pull/198

Reply:
OK, got you. I think.